src/Symfony/Component/Console/Command/Command.php

```diff
@@ -516,7 +516,7 @@ public function getProcessedHelp()
     /**
      * Sets the aliases for the command.
      *
-     * @param string[] $aliases An array of aliases for the command
+     * @param string[]|\Traversable $aliases An array of aliases for the command
```

Comment:
should be reverted, we do not support Traversable here (if that happens to work, fine, but that's not supported).